kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MetadataDoc.java

```diff
@@ -27,20 +25,19 @@ public class MetadataDoc {
   public static final String OFFSET = "offset";
   public static final String EPOCH = "epoch";
 
-  @BsonId
-  ObjectId id;
+  int id;
   int partition;
```

Comment:
@agavra was just wondering, since we're now using the kafka partition as the `id` should we get rid of the redundant `partition` field?  Can you think of any reason to keep it around?
eg: Even if we change/evolve the `id` at some point in the future, that would be a breaking change, so we can always add back the `partition` field at that time.